server/venueless/api/views.py

```diff
@@ -82,6 +83,25 @@ def patch(self, request, **kwargs):
         return Response(serializer.data)
 
 
+class WorldThemeView(APIView):
+
+    permission_classes = []
+
+    def get(self, request, **kwargs):
+        """
+        Retrieve theme config of a world
+        @param request: request obj
+        @param kwargs: world_id
+        @return: theme data of a world
+        """
+        try:
+            world = get_object_or_404(World, id=kwargs["world_id"])
+            return Response(WorldSerializer(world).data['config']['theme'])
+        except KeyError:
```

Comment:
If `KeyError` happens, it means that us, the website owner, did not prepare enough data, or misconfigured the website. We need to do `logger.error` to inform our team.

For end user, we can throw error 503 to tell that this error is our side, temporary and can work again in the future.